chapters/methodology.tex

```diff
@@ -6,7 +6,29 @@ \subsection{User Journey}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
 \subsection{Architecture}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+
+The main goal of the software architecture of soHappy is providing flexibility 
+in switching out various components.
+
+Adding a Face- or Smile-Detector based on the technology of liking requires a

```

Comment:
We should stay consistent here. 😁
I'd write face and smile detector uncapitalised (since it is no name), as well as not using a hyphen